src/cmap/connection_pool.ts

```diff
@@ -107,7 +107,7 @@ export type ConnectionPoolEvents = {
  */
 export class ConnectionPool extends TypedEventEmitter<ConnectionPoolEvents> {
   closed: boolean;
-  options: Readonly<ConnectionPoolOptions>;
+  options: Readonly<ConnectionPoolOptions & { maxConnecting: number }>;
```

Comment:
This is temporary and will move properly into `ConnectionPoolOptions` with the next ticket that makes maxConnecting configurable